### ISSUE TITLE
Support for "Review a user's permission level"

### ISSUE THREAD

**Issue by lmazuel** [PyGithub/PyGithub#722]
Support endpoint:
https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level

I'm not exactly sure what should the accessors:
```python
repo.get_collaborator_permissions("userlogin")
```

Note that it's not the same as listing collaborators and check "permission" node, the string content is not the same.

**Comment by sfdye** [comment]
@lmazuel Indeed, this is missing. Do you want to submit a PR so we can review together? Maybe call it like `review_collaborator_permission`.

**Comment by lmazuel** [comment]
Sure, I patched my own version so I should definitely be able to do a PR. I do that asap.

**Comment by sfdye** [comment]
@lmazuel Any chance to work on the PR?
